modules/update-upgrading-cli.adoc

```diff
@@ -34,49 +34,54 @@ $ oc get clusterversion
 [source,terminal]
 ----
 NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
-version   4.8.13     True        False         158m    Cluster version is 4.8.13
+version   4.8.37     True        False         158m    Cluster version is 4.8.37
 ----
 
-. Review the current update channel information and confirm that your channel
-is set to `stable-4.9`:
+. View the available updates and note the version number of the update that
```

Comment:
The step 2 and step 3 should be switched. We should set channel correctly first, and then it will show available update correctly.